docs/design.md

```diff
@@ -1,14 +1,14 @@
 ## Magic Castle Terraform Structure
 
-![Magic Castle Terraform Structure](https://docs.google.com/drawings/d/e/2PACX-1vSL84MSd1rz5hgjLMfWBBKwp6-jNTEJovNUOXtTwDovpvmyFG7qo12HD32a3fhaKl8hpjcIfM5fZ8II/pub?w=1864&h=1972)
-*Figure 1. Magic Castle Terraform Project Structure*
-
-Figure 1 illustrates how Magic Castle is structured to provide a unified interface between multiple
+Figure 1 (below) illustrates how Magic Castle is structured to provide a unified interface between multiple
 cloud providers. Each blue block is a file or a module, while white blocks are variables or resources.
 Arrows indicate variables or resources that contribute to the definition of the linked variables or
 resources. The figure can be read as a flow-chart from top to bottom. Some resources and variables have been left out of the chart to avoid cluttering it further.
 
-1. `main.tf`: User provides the instances and volumes structure it wants as maps.
+![Magic Castle Terraform Structure](https://docs.google.com/drawings/d/e/2PACX-1vSL84MSd1rz5hgjLMfWBBKwp6-jNTEJovNUOXtTwDovpvmyFG7qo12HD32a3fhaKl8hpjcIfM5fZ8II/pub?w=1864&h=1972)
```

Comment:
I felt it was important to give a bit of context to the figure before introducing it, it's pretty large and takes up most of my screen. I also think you can drop the last blue block in the image since it doesn't seem to contribute anything and is really not mentioned in the rest of the text.

Reply:
Agreed. I fixed the figure.